adoc/deployment-airgapped.adoc

```diff
@@ -866,3 +866,29 @@ Refer to: <<airgap-rpm_repository-client>>.
 === Helm chart connection terminated by HTTPS TO HTTP
 
 When registry mirror is using virtual repository URL. You may need to manually modify the Helm chart index.yaml and point the correct HTTPS base URL.
+
+[#airgap-extras]
+== Extras
+
+=== Check for Deprecated APIs before an Upgrade to New Version in an air gapped deployment
+
+When you want to make sure all the deployments you have are compatible with the new version, in an air gapped environment you have to follow the next steps.
```

Comment:
I would not use the term `deployment` because it can be misleading. kubepug looks for all the types of kubernetes objects defined inside of the cluster. That could include for example ingress definitions.